sormas-ui/src/main/java/de/symeda/sormas/ui/externalmessage/ExternalMessageMapper.java

```diff
@@ -331,14 +331,26 @@ protected ImmutableTriple<String, DiseaseVariant, String> migrateDiseaseVariant(
 		return new ImmutableTriple<>(testResultText, testedDiseaseVariant, testedDiseaseVariantDetails);
 	}
 
-	private FacilityReferenceDto getLabReference(String labExternalId) {
+	public FacilityReferenceDto getLabReference(List<String> labExternalIds) {
+
 		FacilityFacade facilityFacade = FacadeProvider.getFacilityFacade();
-		List<FacilityReferenceDto> labs =
-			labExternalId != null ? facilityFacade.getByExternalIdAndType(labExternalId, FacilityType.LABORATORY, false) : null;
-		if (labs != null && labs.size() == 1) {
-			return labs.get(0);
+		List<FacilityReferenceDto> labs;
+
+		if (labExternalIds != null && !labExternalIds.isEmpty()) {
```

Comment:
After the db migration, I got something like `{NULL}` in the labexternalids column of the for one entry.
This seems to be a list with a null element. This if statement yields true in that scenario, which it probably shouldn't... Or?

Reply:
When trying to process the external message with the `{NULL}` entry, the mapping crashes.

Reply:
@FredrikSchaeferVitagroup can you try again, I use `.filter(Objects::nonNull)` which should remove the object